back-end/API/app/dtos/job_offer_dto.py

```
@@ -0,0 +1,60 @@
+    salary: str = Field(max_length=255)
+    offerDebut: date
+    approbationMessage: str | None = Field(max_length=6000, default=None)
+    enterpriseId: int
+    isApproved: bool | None = False
+    approvedDate: datetime | None = None
+    last_modified_by_id: int | None = None
+    lastModifiedDate: datetime | None = None
+    deadlineApply: date
+ 
+class JobOfferCreateDTO(_JobOfferBase):
+    deadlineApply: date = Field(gt=date.today())
```

Ruff (0.15.13)

[warning] `datetime.date.today()` used

(DTZ011)